packages/next/webpack.config.js

```diff
@@ -18,6 +18,8 @@ const componentWebpackConfig = {
     'payload/config',
     'react-image-crop',
     'payload/operations',
+    'payload/bin',
```

Comment:
i dont think we should be importing from `payload/bin` so this shouldn't be necessary to add. we should try and keep our tweaks to the Webpack config to be as minimal as possible. Ideally we won't have to do anything to it at all